about.html

```diff
@@ -53,59 +53,71 @@
+            subject knowledge for when they return to school.
+            But summer schools often schedule additional workshops and lecture series, which can equip students with
+            interpersonal skills that can help them excel </p>
         <div class="All-Programs">
-            <p class="goals"> Many summer schools are designed to help students improve their academic foundations and
-                help them develop new skills sets for the future.
-                Not only can students immerse themselves in a structured academic programme, helping improve their
-                subject knowledge for when they return to school.
-                But summer schools often schedule additional workshops and lecture series, which can equip students with
-                interpersonal skills that can help them excel in the future workplace. </p>
-            <p class="goals"> Many summer schools are designed to help students improve their academic foundations and
-                help them develop new skills sets for the future.
-                Not only can students immerse themselves in a structured academic programme, helping improve their
-                subject knowledge for when they return to school.
-                But summer schools often schedule additional workshops and lecture series, which can equip students with
-                interpersonal skills that can help them excel in the future workplace.</p>
+            <p class="goals"><img src="/Images/logo1.png" alt=""></p>
         </div>
 
     </div>
 
     <div class="section3">
-        <h1>Our Team</h1>
-
+        <h1>Our Goals For the 2022 Edition</h1>
         <hr class="line" />
-
-        <ul class="Speaker">
-
+        <h2>
+            Many summer schools are designed to help students improve their academic foundations and
+            help them develop new skills sets for the future.
+            Not only can students immerse themselves in a structured academic programme, helping improve their
+            subject knowledge for when they return to school.</h2>
+        <div class="imgs">
+            <ul>
             <li>
                 <div class="Aboutbe4">
+
                     <img src="/Images/aboutback.jpg" alt="personel" class="image">
                     <div class="overlay">
-                        <h2 class="text">2020</h2>
+                        <h2 class="text">2022 Edition</h2>
                     </div>
                 </div>
             </li>
             <li>
                 <div class="Aboutbe4">
                     <img src="/Images/aboutback.jpg" alt="personel" class="image">
                     <div class="overlay">
-                        <h2 class="text">2020</h2>
+                        <h2 class="text">2022 Edition</h2>
                     </div>
                 </div>
             </li>
+        </ul>
+        </div>
 
 
 
-        </ul>
     </div>
 
     <footer>
```

Comment:
- [ ] According to the original design the footer section has a dark (black) background-color on `the Desktop version`,  And there is no `partner section` too, So kindly change the background-color from white to black on `the Desktop version` and remove the `partner section` on desktop version, remember The pages should look almost identical to the original design 👌

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/183268640-dd73171c-7da4-4fc9-a2f0-2a440535470e.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181262424-f012e340-a19c-4492-ab90-d0212098458e.PNG" width="500" /> |